### DISCUSSION TITLE
como colocar cabeçalho e rodapé na apresentação

### DISCUSSION THREAD

**Discussion by jmtannus** [dcTeam23/course#8] [Ideas]
Olá turma! 👍 
Quem quiser colocar cabeçalho e rodapé na apresentação, segue abaixo o exemplo:

header: 'Apresentação: _**nonononon**_'

footer: 'Exercício Markdown - **_Nome do Aluno_** - Digital College - Turma FS23 Aldeota'

**Comment by AbraaoAlves** [comment]
Para saber mais sobre como funciona o esses doi components no Marp, basta acessar: https://marpit.marp.app/directives?id=header-and-footer

Isso fará um header e/ou footer para todos os slides. Caso queira alterar o header/footer de algum slide especifico, basta usar uma diretiva local, como [exemplificado na documentação](https://marpit.marp.app/directives?id=pagination): `<!-- header: true -->`
